### ISSUE TITLE
DuplicatedFinder vs MediaDC

### ISSUE THREAD

**Issue by bonswouar** [PaulLereverend/NextcloudDuplicateFinder#67]
It's not really an issue, feel free to close it if this shouldn't be here!

I guess you've seen this new plugin, MediaDC, that Nextcloud advertised today: https://nextcloud.com/blog/introducing-mediadc-the-app-that-helps-you-find-duplicated-media-files/

I'd like to say I feel like it's a waste of resources to have two repositories doing almost the same thing (as I understand it the main difference being you rely on hashes for all file types and they rely on ffmpeg medias, but in the end they have mostly the same purpose). Although I've noticed @rakekniven is active in both repositories, would you like to share your opinion on that?

Plus I'm worried this new plugin make DuplicateFinder less and less popular/active...

**Comment by rakekniven** [comment]
Hello,

mentioned app is developed by two volunteers. Nextcloud did a blog post about it. Nothing more.
In the open source world it happens many times that you have the choice, see how many note taking apps nextcloud has.

My changes on Nextcloud stuff at GitHub are mostly related to translation stuff. I did not try the other app since today, so I am unable to share any opinion.

**Comment by bonswouar** [comment]
I agree it happens often that you have the choice with OS, but it generally looks like resources (people) are very limited for NC plugins.
Basically adding the "similar image/video but different hash" functionality to DuplicateFinder could have been a great (better?) option, specially in a UX point of view. But as those two volunteers started a new repos they probably wouldn't want that..

**Comment by PaulLereverend** [comment]
Hi 

The apps may look similar but they both have their strengths and weaknesses :
- DuplicateFinder can find duplicates on any types of files but if files are not exactly the same (different hash) they wont be marked as duplicate
- MediaDC (if I understand correctly) can only deal with pictures and videos but is able to find duplicates that are not exactly the same file (different hash)

The apps are compatible so interrested users can install both. Not the best UX but better than nothing 😃

**Comment by bonswouar** [comment]
Thanks for the answers!
Well I'm closing this, but if at some point there is a plan to merge both plugins it would be great ;)
